src/main/java/com/edmunds/tools/databricks/maven/model/JobTemplateModel.java

```diff
@@ -56,23 +55,20 @@ public JobTemplateModel() {
     }
 
     public JobTemplateModel(MavenProject project,
-                            String environment, String databricksRepo, String databricksRepoKey) {
+                            String environment, String databricksRepo, String databricksRepoKey, String groupWithoutCompany) {
```

Comment:
instead of taking groupWithoutCompany, it could also take the "companyPrefix"

JobTemplateModel could then stay the same at the point with the calls to stripCompanyPackage.

The benefits of doing this would be fewer changes overall.